utils/storage/storage.js

```diff
@@ -248,10 +252,45 @@ class Storage {
     );
   }
 
+  fetchAnonymousId(key) {
```

Comment:
Just to make it more readable.
```suggestion
  fetchExternalAnonymousId(key) {
```